bin/convert-argv.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(yargs, argv, convertOptions) {
 	}
 	if(argv.p) {
 		argv["optimize-minimize"] = true;
-		argv["define"] = [].concat(argv["define"] || []).concat("process.env.NODE_ENV=\"production\"");
+		argv["define"] = Array.prototype.concat(argv["define"] || []).concat("process.env.NODE_ENV=\"production\"");
```

Comment:
Hmm while it may be faster, the semantics seem to be different.

Array.prototype.concat('foo') -> ["foo"]
Array.prototype.concat.call('foo') -> [String]

To get the same behavior as `Array.prototype.concat('foo')` you have to do `Array.prototype.concat.call([], 'foo')` which gets us where we started.